<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gestionreservation</class>
 <widget class="QMainWindow" name="gestionreservation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(85, 85, 127);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>258</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:20pt; color:#ffffff;&quot;&gt;Gestion des réservations&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tab_gestionreservations">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>741</width>
      <height>471</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Ajouter une réservation</string>
     </attribute>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>60</y>
        <width>88</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Identifiant Vol:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="idvol">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>50</y>
        <width>551</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableView" name="tableView_vol">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>701</width>
        <height>291</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="ajouterReservation">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>390</y>
        <width>88</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Valider</string>
      </property>
     </widget>
     <widget class="QLabel" name="textSetting">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>4</y>
        <width>331</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Supprimer une réservation</string>
     </attribute>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>40</y>
        <width>88</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Identifiant Vol:</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView_supprev">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>701</width>
        <height>291</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="idvol_2">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>30</y>
        <width>551</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="supprimerRev">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>400</y>
        <width>88</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Valider</string>
      </property>
     </widget>
     <widget class="QLabel" name="testSupp">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>370</y>
        <width>331</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Consulter les réservations</string>
     </attribute>
     <widget class="QTableView" name="tableView_consulterRev">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>701</width>
        <height>381</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>88</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Retour</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
